script-server/README.md

```diff
@@ -56,6 +56,7 @@ Class | Method | HTTP request | Description
 *DefaultApi* | [**runPipeline**](docs/DefaultApi.md#runpipeline) | **POST** /pipeline/{descriptionPath}/run | Run this pipeline
 *DefaultApi* | [**runScript**](docs/DefaultApi.md#runscript) | **POST** /script/{scriptPath}/run | Run this script
 *DefaultApi* | [**scriptListGet**](docs/DefaultApi.md#scriptlistget) | **GET** /script/list | Get a list of available scripts
+*DefaultApi* | [**stopPipeline**](docs/DefaultApi.md#stoppipeline) | **GET** /pipeline/{id}/stop | Stop the specified pipeline run
```

Comment:
Are these endpoints protected in any way ? If yes, can the user only stop their own pipelines ?

Reply:
There is absolutely no protection other that you need to know the exact runId (which is a hash). So it's improbable that you halt someone else's pipeline like this. 
We have no session yet so it's hard to put any kind of protection...